### PR TITLE
Re-add Build Tools for Visual Studio 2022

### PR DESCRIPTION
**Pull request by loic-sharma**
_Description of what this PR is changing or adding, and why:_
Re-add the Build Tools for Visual Studio 2022 option. This is a significantly smaller download than a full Visual Studio installation.

This was originally added by https://github.com/flutter/website/pull/7484 but was removed by https://github.com/flutter/website/pull/9238.

_Issues fixed by this PR (if any):_

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by atsansone** [comment]
Need to check out something with Legal first.

**Review by atsansone — CHANGES_REQUESTED** [review]
Setting to request changes to await Legal advice.

File: src/_includes/docs/install/reqs/windows/software.md [hunk 1]
```diff
@@ -17,10 +17,11 @@
 
 {% else -%}
 
-* [Visual Studio 2022][] to debug and compile native C++ Windows code.
-  Make sure to install the **Desktop development with C++** workload.
-  This enables building Windows app including all of its default components.
-  **Visual Studio** is an IDE separate from **[Visual Studio _Code_][]**.
+* [Visual Studio 2022][] with the the **Desktop development with C++**
```
[the remark this answers is not in the document]
Comment (loic-sharma) [reply src/_includes/docs/install/reqs/windows/software.md]: TIL, updated!

**Review by domesticmouse — APPROVED** [review]

**Comment by sfshaza2** [comment]
What's the status here, @atsansone? I'd like to get this landed or closed, as the case may be.

**Comment by sfshaza2** [comment]
@atsansone, can you remove your "request changes" limitation?

**Comment by atsansone** [comment]
@sfshaza2 : Not yet. I do have my guidance from Legal. I want to check with @anderdobo first. I'll do that today. Sorry for the delay.

**Comment by anderdobo** [comment]
@atsansone I think this is a good option to include if there are no legal or other blockers.

**Review by atsansone — APPROVED** [review]
LGTM. This may need some extra work later, but helps the user now.

**Pull request merged**